shapeshift/web
# feat: fox page internal links

## Description

on top of #2433 
changes external links on fox page opportunities to Defi modals

## Notice




## Pull Request Type


## Issue (if applicable)

closes #2412 

## Risk

none

## Testing

1. enable FoxLp and/or FoxFarming feature flag(s)2. 
2. go to the fox-page3. 
3. CTA on opportunities should be either "Manage" or "Get Started" depending on opportunity balance
4. clicking on CTA should open up the related staking opportunity modal

## Screenshots (if applicable)

<img width="755" alt="Screen Shot 2022-08-15" src="https://user-images.githubusercontent.com/20498757/184618529-d57ca11d-662e-4ff0-ad4a-2824e0dcb381.png">
